frontend/views/pages/Payments.vue

```diff
@@ -279,18 +318,37 @@ export default {
   },
   methods: {
     humanDate,
+    setInitialActiveTab () {
+      this.ephemeral.activeTab = this.needsIncome ? 'PaymentRowReceived' : 'PaymentRowTodo'
+    },
     openModal (name, props) {
       sbp('okTurtles.events/emit', OPEN_MODAL, name, props)
     },
     filterPayment (payment) {
-      const query = this.ephemeral.searchText
+      const query = this.form.search
       const { amount, username, displayName } = payment
       return query === '' || `${amount}${username.toUpperCase()}${displayName.toUpperCase()}`.indexOf(query.toUpperCase()) !== -1
     },
+    paginateList (list) {
+      const start = this.ephemeral.rowsPerPage * this.ephemeral.currentPage
+      return list.slice(start, start + this.ephemeral.rowsPerPage)
+    },
+    handleTabClick (url) {
+      this.ephemeral.activeTab = url
+      this.form.search = ''
```

Comment:
Should changing tabs remove the search string? The user may want to keep their filter active while temporarily checking something out, intending to come back and see it as-is with the search string filtered.

Reply:
That's a nice way of thinking. It makes sense to keep the search indeed

Reply:
Agreed!